src/main/java/mate/academy/rickandmorty/service/implementation/CharacterServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+package mate.academy.rickandmorty.service.implementation;
+
+import jakarta.annotation.PostConstruct;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import mate.academy.rickandmorty.dto.CharactersDtoResponse;
+import mate.academy.rickandmorty.mappers.CharacterMapper;
+import mate.academy.rickandmorty.models.Character;
+import mate.academy.rickandmorty.repository.CharacterRepository;
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class CharacterServiceImpl implements CharacterService {
+    private static final String CHARACTER_URL = "https://rickandmortyapi.com/api/character";
```

Comment:
Suggestion: move to property file. 
```suggestion
    private static final String CHARACTER_URL = "https://rickandmortyapi.com/api/character";
```